update_ip/services/__init__.py

```diff
@@ -1,5 +1,37 @@
-from nfsn import NearlyFreeSpeechService
-from webfaction import WebFactionService
+import inspect
+import os
+import imp
+import glob
+from base import BaseDNSService
 
-services= [WebFactionService, NearlyFreeSpeechService]
```

Comment:
Personally, I like the current hardcoded services.
This adds a lot of nonn-trivial code that is hard to test (assuming we'll ever have unit tests...), and removes the option of (easily) customizing which services to load.
A middle way would be accepting a list of strings, instead of a list of modules.
@bkonkle Do you have a preference?